lighthouse-core/aggregator/aggregate.js

```diff
@@ -77,6 +77,12 @@ class Aggregate {
       throw new Error(msg);
     }
 
+    // Audit resulted in an error, so doesn't contribute to score.
```

Comment:
@brendankenny and I talked this out. I think a "less complete" score that factors in fewer audits makes more sense than seeing `NaN` for the entire category. Basically, users should never see `NaN`. That looks like a bug.